src/component/marker/MarkAreaView.ts

```diff
@@ -248,22 +248,19 @@ class MarkAreaView extends MarkerView {
             const points = map(dimPermutations, function (dim) {
                 return getSingleMarkerEndPoint(areaData, idx, dim, seriesModel, api);
             });
+            const xAxis = coordSys.getAxis('x').scale.getExtent();
+            const yAxis = coordSys.getAxis('y').scale.getExtent();
+            const xPoint = [coordSys.getAxis('x').scale.parse(areaData.get('x0', idx)),
```

Comment:
It's better to 'cache' `coordSys.getAxis('x').scale` and `coordSys.getAxis('y').scale`

Reply:
thanks for your advice :-)